HitDelay.py

```
@@ -1,429 +1,429 @@
+			self.historyDelayValueLabel['text'] = '%.6fms  '%float(data[2])
+			self.historyAccValueLabel['text'] = '%.6fms  '%float(data[4])
+			del data
+
+	def DeleteCursorHistory(self):
+		self.__logger.info(f"delete history {self.cursorHistory}")
+		if self.cursorHistory == '':
+			messagebox.showerror('Error','请输入谱面标识')
+		else:
+			result = messagebox.askyesno('提示', f'是否删除该谱面游玩记录?\n{self.cursorHistory}')
+			if result:
+				self.cur.execute(f'delete from HitDelayHistory where SongMapName=\'{self.cursorHistory}\' and RecordTime=\'{self.historyRecordTimeValueLabel["text"]}\'')
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)